src/systems/particle_emitter/ParticleEmitter.hh

```diff
@@ -37,6 +37,8 @@ namespace systems
   ///
   /// `<emitter_name>`: Unique name for the particle emitter. The name will be
   ///                   automatically generated if this parameter is not set.
+  /// `<allow_renaming>`: Rename the particle emitter if one with the same name
+  ///                     already exists
```

Comment:
We should also mention that this is argument is of `boolean` type and has a default value of `false`.

Reply:
ff65cb6